Userland/Libraries/LibWeb/Layout/TextNode.cpp

```diff
@@ -357,18 +357,29 @@ void TextNode::compute_text_for_rendering()
         return;
     }
 
+    auto do_ellipsis = computed_values().text_overflow() == CSS::TextOverflow::Ellipsis
```

Comment:
Commit message nit-pick: You used the word "ellipse" a couple of times when you mean "ellipsis".